OpenSim/Simulation/Test/testStationDefinedFrame.cpp

```diff
@@ -273,9 +273,9 @@ TEST_CASE("StationDefinedFrame_SanityCheckCanUsePOFAsParentOfJointViaOtherOffset
     // isn't handled correctly by OpenSim's graph traversal
     //
     // remove the `REQUIRE` part and uncomment the other lines if you think you've fixed this
-    REQUIRE_THROWS(model.buildSystem());
-    // SimTK::State state = model.initializeState();
-    // model.realizeReport(state);
+    model.buildSystem();
```

Comment:
This test was put into the `StationDefinedFrame` feature as a smoke test to verify that a known bug still exists. The bug is fixed by this PR, so the test was reenabled.